src/common/SurgeSynthesizer.cpp

```diff
@@ -2086,6 +2086,17 @@ void SurgeSynthesizer::setSamplerate(float sr)
             f->sampleRateReset();
         }
     }
+
+    for (int s = 0; s < n_scenes; s++)
+    {
+        auto iter = voices[s].begin();
+        while (iter != voices[s].end())
+        {
+            SurgeVoice *v = *iter;
+            assert(v);
+            v->sampleRateReset();
+        }
+    }
```

Comment:
This is fine of course but does surge work if we change sr with playing voices? I guess now it won’t break here :)